lib/ts/recipe/session/index.ts

```diff
@@ -46,6 +46,7 @@ export default class SessionWrapper {
         recipeUserId: RecipeUserId,
         accessTokenPayload: any = {},
         sessionDataInDatabase: any = {},
+        alwaysOverwriteSessionInRequest: boolean = true,
```

Comment:
Can this be an issue because `userContext` is typed as `any`? (Meaning if the user passed an object typed as any it'll pass type checking)
We do not necessarily need to change this now, but it's worth thinking about.

Reply:
Yea I understand, but couldn't think of better way to add it